gitless/tests/utils.py

```diff
@@ -82,6 +82,16 @@ def onerror(func, path, unused_exc_info):  # error handler for rmtree
   logging.debug('Removed dir {0}'.format(path))
 
 
+def symlink(src, dst, target_is_directory=False, *, dir_fd=None):
```

Comment:
is the `*` useful for something in this case? It doesn't seem like you are using it and it breaks the python 2 tests